src/StreetNameRegistry.Api.BackOffice/Infrastructure/Authorization/IRegistryNisCodeFinder.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace StreetNameRegistry.Api.BackOffice.Infrastructure.Authorization
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public interface IRegistryNisCodeFinder<T>
```

Comment:
I'm having trouble with this name. This is streetname registry and we already have 2 finders. `MunicipalityRegistryFinder` seems not really a good name.